pkg/flowaggregator/s3uploader/s3uploader_test.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package s3uploader
+
+import (
+	"bytes"
+	"context"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/config"
+	s3manager "github.com/aws/aws-sdk-go-v2/feature/s3/manager"
+	s3service "github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/gammazero/deque"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+	ipfixentitiestesting "github.com/vmware/go-ipfix/pkg/entities/testing"
+	"github.com/vmware/go-ipfix/pkg/registry"
+
+	"antrea.io/antrea/pkg/flowaggregator/flowrecord"
+)
+
+var buf bytes.Buffer
+
+// Skip the string matching for timestamps, as the time.Format() method gives
+// different results according to local time zone.
```

Comment:
should we just use UTC timestamps everywhere then? Otherwise it feels like a problem for whoever ends up consuming the CSV files as well.